Source/CombatExtended/Compatibility/VanillaPsycastExpanded.cs

```diff
@@ -76,10 +76,13 @@ public static bool AOE_CheckIntercept(ProjectileCE projectile, Vector3 from, Vec
                 .SelectMany(x => x.health.hediffSet.hediffs)
                 .Where(x => x is Hediff_Overshield && x.GetType() != typeof(Hediff_Overshield)).Cast<Hediff_Overshield>())
             {
-                Vector3 shieldPosition = interceptor.pawn.Position.ToVector3ShiftedWithAltitude(0.5f);
+                Vector3 shieldPosition = interceptor.pawn.Position.ToVector3Shifted().Yto0();
                 float radius = interceptor.OverlaySize;
-                float blockRadius = radius + def.projectile.SpeedTilesPerTick + 0.1f;
-                if (CE_Utility.IntersectionPoint(from, newExactPos, shieldPosition, radius, out Vector3[] sect))
+                if ((new Vector3(projectile.origin.x, 0, projectile.origin.y) - shieldPosition).sqrMagnitude < radius * radius)
```

Comment:
this could use a comment that it is to handle outgoing projectiles

Reply:
Done.